legend-engine-language-pure-compiler/src/main/java/org/finos/legend/engine/language/pure/compiler/toPureGraph/PureModel.java

```diff
@@ -294,6 +299,11 @@ public PureModel(PureModelContextData pureModelContextData, Iterable<? extends C
         }
     }
 
+    public static PureModel getSystemPureModel()
```

Comment:
@gs-ssh16 @pierredebelen what do you think of this util function, it helps us use constants used in pure as source of truth